template.yml

```diff
@@ -21,34 +21,10 @@ Resources:
       ApplicationName: !Ref application
       EnvironmentName: BETA
       OptionSettings:
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: AWS_REGION
-          Value: !Ref AWS::Region
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: NOTIFICATION_TOPIC
-          Value: !Ref notificationTopic
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: NOTIFICATION_EMAIL
-          Value: !Ref emailAddress
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: USER_TABLE
-          Value: !Ref userTable
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: SESSION_TABLE
```

Comment:
These environment variables are used by the application to identify tables, instead of using hardcoded table names. What happened when you tried using the web app?

Reply:
When David and I tested the web app, it worked fine with this change and there were no errors.
I see that by default the app will use the DDB table name defined by the [annotation](https://github.com/aws-samples/eb-java-scorekeep/blob/edda9475438a62d503470d40300bae870fd332b0/src/main/java/scorekeep/Session.java#L11) which is actually [hard-coded in the TableNames class](https://github.com/aws-samples/eb-java-scorekeep/blob/edda9475438a62d503470d40300bae870fd332b0/src/main/java/scorekeep/TableNames.java#L5).

I realized that the environment variables like `SESSION_TABLE` were to enable users to [override](https://github.com/aws-samples/eb-java-scorekeep/blob/edda9475438a62d503470d40300bae870fd332b0/src/main/java/scorekeep/SessionModel.java#L17) the hard-coded table names. I had to drop this ability in favor of being able to run the app both via the EB console and the shell scripts.

Reply:
Thanks for confirming. Curious that `TableNameOverride` doesn't throw an error with a null table name. I'll update the `4-run-local.sh` script to not look for these values in the stack.

Reply:
Yeah, I wondered the same. I haven't tested it but [this doc](https://docs.aws.amazon.com/en_us/amazondynamodb/latest/developerguide/DynamoDBMapper.OptionalConfig.html) shows that the `TableNameOverride` can actually handle null.